.github/workflows/release.yml

```diff
@@ -24,6 +24,26 @@ jobs:
         with:
           go-version: "1.18.x"
 
+      - name: Build tagged release, elf, debug
+        id: build-debug-elf
+        uses: oasisprotocol/oasis-sdk/.github/actions/hash-rust@main
+        with:
+          image: oasisprotocol/runtime-builder:main
+          pkg-dirs: runtime
+          binaries: emerald-paratime
+          clean: no
+          setup: |
+            export OASIS_UNSAFE_USE_LOCALNET_CHAINID=1
```

Comment:
From the variable name I would expect the following usage `OASIS_UNSAFE_USE_LOCALNET_CHAINID=0xa514`. But then this could also be reused for building official Testnet and Mainnet variants...

Reply:
There are bunch of issues around this:
1. `CHAIN_ID` is constant and can only be set at [compile time](https://github.com/oasisprotocol/oasis-sdk/blob/main/runtime-sdk/modules/evm/src/lib.rs#L57).
2. And now since it needs to be hardcoded at compile time, up to date rust has issues with [parsing env variables to integers](https://github.com/rust-lang/rfcs/issues/1907).
3. We could probably use konst (see the [last comment](https://github.com/rust-lang/rfcs/issues/1907#issuecomment-808646667)), but it smells wrong.